package.json

```diff
@@ -8,16 +8,21 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "async": "^3.2.3",
```

Comment:
What is this used for? If you're using `async/await` syntax, there is no module for that; it's a part of JavaScript

Reply:
It might be something wrong with my local machine, at some point I have got an error message that there is no async, and installed it so I could continue.